src/tic_tac_toe/display/terminal.clj

```diff
@@ -3,30 +3,51 @@
             [tic-tac-toe.game :refer [winner?]]
             [tic-tac-toe.protocol.display :refer [DisplayProtocol]]))
 
+(def base-index 0)
+
+(def marker-a "x")
+(def marker-b "o")
+
+(def left-column "| ")
+(def small-right-column " |")
+(def large-right-column "  |")
+
 (defn clear-terminal[]
   (println "\033[2J"))
 
 (defn print-message [message]
   (println message))
 
-(defn display-index [board]
-  (let [game-index (partition (count board) (range 1 (+ 1 (board-size board))))]
-    (println "Game Index")
-    (doseq [row game-index] (println (apply str row)))))
+(defn empty-position [element]
```

Comment:
What does `element` refer to in this context? Can you think of a more descriptive name?